File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/VpsDALicense.scala
package io.swagger.server.model


/**
 * A DirectAdmin license tier option.
 *
 * @param name License tier name.
 * @param sub_name License tier sub-name.
 * @param cost Monthly cost in cents.
 * @param img_disabled Image path for the disabled state icon.
 * @param img_active Image path for the active state icon.
 */
case class VpsDALicense (
  name: Option[String],
  sub_name: Option[String],
  cost: Option[Int],
  img_disabled: Option[String],
  img_active: Option[String]
)

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/VpsDALicense.js
goog.provide('API.Client.VpsDALicense');

/**
 * A DirectAdmin license tier option.
 * @record
 */
API.Client.VpsDALicense = function() {}

/**
 * License tier name.
 * @type {!string}
 * @export
 */
API.Client.VpsDALicense.prototype.name;

/**
 * License tier sub-name.
 * @type {!string}
 * @export
 */
API.Client.VpsDALicense.prototype.subName;

/**
 * Monthly cost in cents.
 * @type {!number}
 * @export
 */
API.Client.VpsDALicense.prototype.cost;

/**
 * Image path for the disabled state icon.
 * @type {!string}
 * @export
 */
API.Client.VpsDALicense.prototype.imgDisabled;

/**
 * Image path for the active state icon.
 * @type {!string}
 * @export
 */
API.Client.VpsDALicense.prototype.imgActive;

